hexrdgui/image_canvas.py

```diff
@@ -1034,10 +1039,39 @@ def finish_show_polar(self, iviewer):
                 }
                 self.axes_images.append(self.axis.imshow(**kwargs))
                 self.axis.axis('auto')
+
+                self.axis.yaxis.set_major_locator(AutoLocator())
+                self.axis.yaxis.set_minor_locator(AutoMinorLocator())
+
+                self.axis.xaxis.set_major_locator(PolarXAxisTickLocator(self))
+                self.axis.xaxis.set_minor_locator(PolarXAxisMinorTickLocator(self))
+
+                kwargs = {
+                    'left': True,
+                    'right': True,
+                    'bottom': True,
+                    'top': True,
+                    'which': 'major',
+                    'length': 10,
+                    'labelfontfamily': 'serif',
+                    'labelsize': FONTSIZE_TICKS,
+                }
+                self.axis.tick_params(**kwargs)
+
+                kwargs['which'] = 'minor'
+                kwargs['length'] = 2
+                self.axis.tick_params(**kwargs)
+
+                self.axis.tick_params(
+                   bottom=True, top=True, which='major', length=8)
+                self.axis.tick_params(
+                    bottom=True, top=True, which='minor', length=2)
```

Comment:
I believe these don't have have an effect (these appear to be set by the previous calls to `tick_params`, and I don't see a difference in the plots if I remove these lines).

Do you know if these lines are necessary?